scripts/background.js

```diff
@@ -3,20 +3,22 @@ chrome.tabs.onUpdated.addListener((tabId, changeInfo, tab) => {
   if (
     changeInfo.status === "complete" &&
     tabUrl &&
-    tabUrl.includes("linkedin.com/jobs/search")
+    tabUrl.includes("linkedin.com/jobs/view")
```

Comment:
Moved this to the top since `/jobs` is part of `jobs/views` so we would've never gotten into this condition if it were in the `else-if` block.